routes/Hierarchy/controllers.js

```diff
@@ -14,19 +14,15 @@ const controller = {
     }
   },
 
- get: async (req, res) => {
-    try{
-       //*passing the req
-      const addResponse = await hierarchy.get(req.params.id);
-      if (addResponse.message === "success") {
-        return httpResponse.SUCCESS(res, addResponse);
-      } else {
-        return httpResponse.UNAUTHORIZED(res, addResponse.data);
-      }
+  get: async (req, res) => {
+    const addResponse = await hierarchy.get(req.params.id);
+    if (addResponse.message === "success") {
+      return httpResponse.CREATED(res, addResponse);

```

Comment:
Status code for each request?

Reply:
sir i'm actually confused what you're to say here like shouldn't i use http response for every type of failure or should i use for every type kindly i'd be better if you'd elaborate